crates/rooch-server/src/server/rooch_server.rs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) RoochNetwork
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::api::rooch_api::RoochAPIServer;
+use crate::api::RoochRpcModule;
+use crate::response::JsonResponse;
+use crate::service::RpcService;
+use ethers::types::Bytes;
+use jsonrpsee::{
+    core::{async_trait, RpcResult},
+    RpcModule,
+};
+use move_core_types::{
+    account_address::AccountAddress,
+    identifier::Identifier,
+    language_storage::{ModuleId, TypeTag},
+};
+use moveos::moveos::TransactionOutput;
+use moveos_types::{object::ObjectID, transaction::AuthenticatableTransaction};
+use rooch_types::transaction::TypedTransaction;
+use rooch_types::{transaction::rooch::RoochTransaction, H256};
+
+pub struct RoochServer {
+    rpc_service: RpcService,
+}
+
+impl RoochServer {
+    pub fn new(rpc_service: RpcService) -> Self {
+        Self { rpc_service }
+    }
+}
+
+#[async_trait]
+impl RoochAPIServer for RoochServer {
```

Comment:
Default define rooch_,wallet_,eth_ three namespace, later, as more interfaces are implemented, these corresponding `xxxServer` files may be relatively large and difficult to split